swagger/index.js

```diff
@@ -131,13 +133,21 @@ module.exports = yeoman.Base.extend({
           name: model.name,
           plural: model.plural,
           base: model.base || 'PersistedModel',
-          facetName: 'server', // hard-coded for now
+          facetName: 'common', // hard-coded for now
```

Comment:
The new swagger->loopback mapping will be released as a new major. It significantly changes the convention. We'll update the document to reflect that. There is not much migration we can do though.